cms/templates/widgets/sock.html

```diff
@@ -23,43 +23,59 @@ <h2 class="title sr">${_("{studio_name} Documentation").format(studio_name=setti
         <h3 class="title">${_("{studio_name} Documentation").format(studio_name=settings.STUDIO_NAME)}</h3>
 
         <div class="copy">
-           <p>${_("You can click Help in the upper right corner of any page to get more information about the page you're on. You can also use the links below to download the Building and Running an {platform_name} Course PDF file, to go to the {platform_name} Author Support site, or to enroll in edX101.").format(platform_name=settings.PLATFORM_NAME)}</p>
+          <p>${_("You can click Help in the upper right corner of any page to get more information about the page you're on. You can also use the links below to download the Building and Running an {platform_name} Course PDF file, to go to the {platform_name} Author Support site, or to enroll in edX101.").format(platform_name=settings.PLATFORM_NAME)}</p>
```

Comment:
@explorerleslie @peter-fogg Not addressed explicitly in the JIRA story, but we should also update this text too. OR, do you think we can do without it?
The "Studio Documentation" heading above is also no longer applicable.